.gitmodules

```diff
@@ -1,6 +1,9 @@
 [submodule "third_party/netlistsvg"]
 	path = third_party/netlistsvg
 	url = https://github.com/nturley/netlistsvg.git
+[submodule "third_party/prjxray"]
+	path = third_party/prjxray
+	url = https://github.com/SymbiFlow/prjxray.git
 [submodule "third_party/prjxray-db"]
 	path = third_party/prjxray-db
```

Comment:
Maybe we should move `third_party/prjxray-db` to `third_party/prjxray/database` now?

Let's just log and issue and worry about it later :-P